### PR TITLE
init npm

### PR DESCRIPTION
**Pull request by tseabra-rocket**
Added package.json to use with npm install

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "the-modal",
+  "version": "1.0.0",
+  "description": "The Modal =========",
```
Comment (samdark) [inline package.json]: Why `=========`?

Comment (tseabra-rocket) [inline package.json]: It was what npm write.

**Comment by tseabra-rocket** [comment]
options are needed for onClose to work

**Comment by samdark** [comment]
Thanks!

**Pull request merged**